### ISSUE TITLE
I have prometheus as backend, I use netdata's GUI instead of grafana for long term data?

### ISSUE THREAD

**Issue by medyagh** [firehol/netdata#3112]
I like netdata's GUI, I have installed netdata on a few nodes and I have configured prometheus to scrape them. I know some people say I should use Grafana on top of prometheus to see dashboards.

is there any way I could use Netdata's GUI ? I like its gui and its charts?
if not, is there any way to export all the dashboards of netdata to grafana without me recreating them?

**Comment by ktsaou** [comment]
I am very sorry. The answer is no to both questions. netdata dashboard can only use a netdata as backend (actually, the backend has to be really fast for the dashboard to function, and such a speed is not possible for any other backend - netdata responds in less than 1 ms to most queries).

Regarding the netdata dashboards at grafana, I have seen a few attempts to create basic system monitoring dashboards. Check the grafana site, I think they have a registry of templates.

However, netdata monitors way too many apps, so don't expect something that would be as complete as the netdata dashboards are.

**Comment by ktsaou** [comment]
Of course, if you manage to work something out, I would be grateful if you contribute it back to the community.
